BUILD

```diff
@@ -32,3 +32,10 @@ genrule(
     cmd = "echo $${ISTIO_VERSION:-\"0.3.0-dev\"} > \"$@\"",
     visibility = ["//visibility:public"],
 )
+
+# Shell script added for OSSM-1931
+sh_binary(
```

Comment:
Why do we need this? Why don't we have this in maistra/ci/common.sh?

Reply:
Yes, this can be put in common.sh